Datasets/WordSeg/WordSegDataset.swift

```diff
@@ -15,28 +15,51 @@
 import Foundation
 import ModelSupport
 
+/// A collection of raw and processed text used for training and validation
+/// of word segmentation models.
 public struct WordSegDataset {
+  /// A collection of text used for training.
```

Comment:
```suggestion
  /// The text used for training.
```

Saying “a collection” is redunant here; we know it's an array by looking at the declaration.

Reply:
Applied this change